charts/pega/values-large.yaml

```diff
@@ -214,6 +215,7 @@ global:
 
       hpa:
         enabled: true
```

Comment:
Why does the stream tier have scaling set to true at all?  Should be false to match values.yaml right?

Reply:
@dcasavant I don't think this is a question for me, this story was about making it possible to remove one of the targets in HPA and do this for dataflow tiers. The stream tier is outside the scope of this story and it's a different team that owns it.